### ISSUE TITLE
Return extraction progress in AP /status call

### ISSUE THREAD

**Issue by jgaff** [xtracthub/xtracthub-service#27]
It looks like the progress of an extraction task isn't returned from the Action Provider /status call. Having information like the number of groups/families extracted out of the total allows users to see that the extraction is ongoing, and also estimate how long the process may take, and also allows for some basic sanity-checking. Can this be added somewhat easily?

**Comment by tskluzac** [comment]
Yes, we have that information available from the crawler so wouldn't be too hard to update the extraction service itself to do the same. I'll see what that would entail and outline it here.
